R/data_summary.R

```diff
@@ -122,7 +122,7 @@ data_summary.data.frame <- function(x, ..., by = NULL, remove_na = FALSE) {
       # bind grouping-variables and values
       summarised_data <- cbind(s[1, by], summarised_data)
       # make sure we have proper column names
-      colnames(summarised_data) <- c(by, vapply(summarise, names, character(1)))
+      colnames(summarised_data) <- c(by, unlist(lapply(summarise, names)))
```

Comment:
Is this a key change for this PR or just a cosmetic one? If the latter, I'd rather keep the `vapply()` call which ensures that we get the right output type.

Reply:
It's because `bayestestR::ci()` may return more than one value, which would fail with `character(1)`.